.github/CODEOWNERS

```
@@ -631,6 +631,9 @@ playwright.config.ts @grafana/plugins-platform-frontend
 # SSE - Server Side Expressions
 /pkg/expr/ @grafana/observability-metrics
```

Just change this instead

```suggestion
/pkg/expr/ @grafana/grafana-datasources-core-services